src/concept/mod.rs

```diff
@@ -44,6 +44,22 @@ pub enum Concept {
     Attribute(Attribute),
 }
 
+impl Concept {
+    pub fn type_label_cloned(&self) -> String {
+        // FIXME: Add a Label type to simplify this function
+        match self {
+            Concept::RoleType(RoleType { label, .. }) => format!("{label}"),
+            Concept::RootThingType(_) => RootThingType::LABEL.to_string(),
```

Comment:
`.to_string()` implies formatting, i.e. it noramlly calls `Display::fmt`. While I'm pretty sure there's a special case for `&str`, `.to_owned()` to me still seems clearer in intent.